pkg/controllers/edgeconnect/controller.go

```diff
@@ -447,12 +447,12 @@ func (controller *Controller) reconcileEdgeConnectProvisioner(ctx context.Contex
 		}
 	}
 
-	k8sHostname := controller.k8sAutomationHostPattern(edgeConnect.Name, edgeConnect.Namespace, edgeConnect.Status.KubeSystemUID)
+	k8sAutomationHostPattern := controller.k8sAutomationHostPattern(edgeConnect.Name, edgeConnect.Namespace, edgeConnect.Status.KubeSystemUID)
 
-	hostPatterns := controller.hostPatterns(edgeConnect, k8sHostname)
+	hostPatterns := controller.hostPatterns(edgeConnect, k8sAutomationHostPattern)
```

Comment:
(I know this is more of a comment for the previous change 😬 )

Why isn't `hostPatterns` a method on the `EdgeConnect` type ?

(you are only using parts of `EdgeConnect` so it would be a nice abstraction)
(also, having it as a method of the `controller` makes little sense, when it doesn't use anything from the `controller`)

Reply:
Done [here](https://github.com/Dynatrace/dynatrace-operator/pull/3332/commits/fbcf1463cce08347a2eec61a87270e2e658af52c)